### ISSUE TITLE
Chore commit type

### ISSUE THREAD

**Issue by aleksator** [conventional-commits/conventionalcommits.org#303]
What do you think of not recommending a `chore` commit type as it was removed from Angular Convention in [2017](https://github.com/angular/angular/commit/dff6ee32725197bdb81f3f63c5bd9805f2ed22bb#diff-6a3371457528722a734f3c51d9238c13)?

As an alternative, defining a specific meaning for this (and others) commit type would be IMO a great addition to the spec.

**Comment by ciscoski** [comment]
I fully agree.
The meaning of the tag should be specified or at least a subset of the specification.

**Comment by nedbat** [comment]
FWIW, I've defined it as "a repetitive mechanical task such as updating requirements or translations."

**Comment by tianzhich** [comment]
Is the `chore` replaced by `ci` and `build`? I saw some [articles](https://nitayneeman.com/posts/understanding-semantic-commit-messages-using-git-and-angular/#build) or [comments](https://gist.github.com/brianclements/841ea7bffdb01346392c#gistcomment-3037243) that said this. But from [this commit](https://github.com/angular/angular/commit/dff6ee32725197bdb81f3f63c5bd9805f2ed22bb#diff-6a3371457528722a734f3c51d9238c13) angular delete the `chore` type as it was overused, but not prepare to replace it with `ci` and `build`.

**Comment by daiyam** [comment]
I feel like `chore` is kinda too generic. I've replaced it with `ci`, `build`, `vcs` and `enhance`.

@tianzhich What's your usage?

**Comment by tianzhich** [comment]
> I feel like `chore` is kinda too generic. I've replaced it with `ci`, `build`, `vcs` and `enhance`.
> 
> @tianzhich What's your usage?

Yes - It's too generic so different teams use it in different ways. There are two teams in my company department, one of them use it like `ci` or `build`, another use it just like this words meaning *some bothering little changes*. IMO the conventional-commits team could remove it like Angular team or replaced it with some more specific words like what you did.

**Comment by javier-godoy** [comment]
We had the same discussion in our team (https://github.com/FlowingCode/DevelopmentConventions/issues/11). Since we are already using other types from the Angular convention (`build:`, `ci:`, etc) in addition to `feat:` and `fix:`, we settled on defining `chore:` as 
> Changes, not covered by other types

In practice, that means that `chore:` is used for changes to .gitignore, license headers, and static resources such as images.

**Comment by damianopetrungaro** [comment]
I see this issue having some 🔔 , what do you think @bcoe ? 

Should we drop the support? 

Maybe waiting for some more community to ask for this change?

**Comment by javier-godoy** [comment]
@damianopetrungaro Is there anything to be dropped? In strict sense, conventional-commits just specify that "_types other than `fix:` and `feat:` are allowed_". The mention of `chore:` is just by means of example.

Then, it's not an issue with Conventional Commits, but with the actual definition of "_types other than `fix:` and `feat:`_" as used by a specific community of practice. The definition adopted by the old Angular Convention ("_other changes that don't modify src or test files"_) was too wide, but other definitions don't pose this problem: for instance, [AngularJS](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#type) defines it as "_changes to the build process or auxiliary tools and libraries such as documentation generation_" which is mostly equivalent to the `build:` and `ci:` types used by other teams.
